sdk/appconfiguration/azure-data-appconfiguration/src/main/java/module-info.java

```diff
@@ -3,6 +3,8 @@
 
 module com.azure.data.appconfiguration {
     requires transitive com.azure.core;
+    requires transitive com.azure.http.netty;
```

Comment:
Netty should not be added as the user can opt to use a different HTTP client.

Reply:
will revert the changes